client-misk/src/main/kotlin/app/cash/backfila/client/misk/BackfillConfig.kt

```diff
@@ -3,6 +3,15 @@ package app.cash.backfila.client.misk
 import okio.ByteString
 
 data class BackfillConfig(
+  /**
+   * Populated from class level @Parameter annotations
+   * Use parameter function to get just the value (as a String)
+   * @see com.squareup.backfila.client.Parameter
+   */
   val parameters: Map<String, ByteString>,
   val dryRun: Boolean = true
-)
+) {
+  fun parameter(name: String): String? {
```

Comment:
nit: `getParameterUtf8` or `getParameter` would be a bit more descriptive function name